miden-tx/src/host/mod.rs

```diff
@@ -105,12 +105,15 @@ impl<A: AdviceProvider, T: TransactionAuthenticator> TransactionHost<A, T> {
     // EVENT HANDLERS
     // --------------------------------------------------------------------------------------------
 
-    fn on_note_created<S: ProcessState>(
+    /// Creates a new note from the pointer and stores it in the `output_notes` field of the
+    /// [TransactionHost].
```

Comment:
nit: I would probably say: "Crates a new [OutputNoteBuilder] from the data on the operand stack and stores it into the `output_notes` field of this [TransactionHost]."